sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -238,6 +242,172 @@ describe(describeProcessorRootTest(QueueNames.SFASIntegration), () => {
     },
   );
 
+  it(
+    "Should not add SFAS individual data records and display errors when importing invalid data from SFAS " +
```

Comment:
`and display errors` can be removed from the description. 
![image](https://github.com/user-attachments/assets/6a91b906-8827-4594-800a-ed04ec4fd33c)